tests/common/helpers/redis.py

```diff
@@ -311,12 +311,9 @@ def get_neighbor_value(self, neighbor_key, field):
             neighbor_key: The full key of the neighbor table.
             field: The field to get in the neighbor hash table.
         """
-        cmd = ["/usr/bin/redis-cli", "-n", "1", "HGET", neighbor_key, field]
-        if self.host.namespace is not None:
-            cmd = ["/usr/bin/redis-cli", "-h", str(self.ip), "-n", "1", "HGET", neighbor_key, field]
-            cmd = ["sudo", "ip", "netns", "exec"] + [self.host.namespace] + cmd
+        cmd = "%s ASIC_DB HGET '%s' %s" % (self.host.sonic_db_cli, neighbor_key, field)
```

Comment:
do we need the -n <namespace> for sonic_db_cli as well ?

Reply:
Answered above, sonic_db_cli attribute includes the namespace.